models/appointment.model.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * this file will contain the appointment resources atributes
+ */
+
+ const mongoose = require("mongoose");
+ const constants = require("../utils/constants");
+
+ const appointmentSchema = new mongoose.Schema({ 
+           /**
+            *  appointmentId, docId, patientId, timing, createdAt, updatedAt 
+            */       
+    docId : {
+        type : String,
```

Comment:
Better to use an Objectid as reference.

Same comment applies at the other places as well